ibm/mas_devops/roles/ocp_deprovision/tasks/providers/fyre.yml

```diff
@@ -16,12 +16,13 @@
       - "Cluster name ................. {{ cluster_name }}"
       - "Username ..................... {{ fyre_username }}"
       - "Password ..................... *****************"
+      - "Site ......................... {{ fyre_site }}"
 
 # 2. Determine whether cluster exists
 # -----------------------------------------------------------------------------
 - name: "quickburn : Check if cluster already exists"
   uri:
-    url: "https://ocpapi.svl.ibm.com/v1/check_hostname/{{ cluster_name }}"
+    url: "https://ocpapi.svl.ibm.com/v1/check_hostname/{{ cluster_name }}?site={{ fyre_site }}"
```

Comment:
Site may not be necessary for check_hostname.

Reply:
@Aymen-Ben-S I thought the same till the guy from Fyre suggested the approach above for GETS and use the side in the payload for POST and PUT. It seems svl url is the same regardless the side we want to provision.